Modules/Messages/test/Messages.API.Tests.Performance.EndToEnd/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
es5 is from 2009, so it's very old. Is it possible to use the latest version (es2022)?